bcipy/signal/processing/decomposition/psd.py

```diff
@@ -0,0 +1,80 @@
+import matplotlib.pyplot as plt
+import seaborn as sns
+import numpy as np
+from enum import Enum
+from scipy import signal
+from scipy.signal import welch
+from scipy.integrate import simps
+from mne.time_frequency import psd_array_multitaper
+
+
+class PSD_TYPE(Enum):
+    WELCH = 'Welch'
+    MULTITAPER = 'MutliTaper'
+
+
+def power_spectral_density(
+        data: np.ndarray,
+        band: tuple,
+        fs: float=100,
+        window_length: float=4,
+        plot: bool=False,
+        method: PSD_TYPE=PSD_TYPE.MULTITAPER,
+        relative=False):
+    """Power spectral density:
+
+    Many thanks to: https://raphaelvallat.github.io/bandpower.html
```

Comment:
It might be more clear to name the parameter window_seconds, or to provide parameter documentation. Also, it seems like the types of the default arguments should match the declared variable types (ex. 4.0).